scss/_patterns_lists.scss

```diff
@@ -133,11 +135,45 @@ $list-status-icon-nudge: 0.3rem; // 0.3rem seems to be a magic number that posit
     .p-list__item {
       @extend %vf-list-item;
       @include vf-list-item-divided;
+
+      &.nested-list-wrapper {
+        box-shadow: none;
+        padding-bottom: 0;
+        padding-top: 0;
+
+        &::before {
+          display: none;
+        }
+      }
+
+      // the :has fallback has to be in a separate declaration with dupplicated rules
+      &:has(ol),
+      &:has(ul) {
+        box-shadow: none;
+        padding-bottom: 0;
+        padding-top: 0;
+
+        &::before {
+          display: none;
+        }
+      }
```

Comment:
Something funky is happening here:
If I put it all in the same section like this
```css
      &:has(ol),
      &:has(ul), 
      &.nested-list-wrapper {
        box-shadow: none;
        padding-bottom: 0;
        padding-top: 0;

        &::before {
          display: none;
        }
      }
```

Then the rule gets ignored by browsers that don't support the `:has()` pseudo selector.

I also tried 

```css
      &.nested-list-wrapper {
         @extends %some-placeholder
      }

      &:has(ol),
      &:has(ul) {
        @extends %some-placeholder
      }
```

But for some reason I don't understand, it gets ignored too. I suspect it's in the way placeholders are compiled